lib/rucio/client/baseclient.py

```diff
@@ -114,6 +115,13 @@
     expiration_time=60,
 )
 
+STATUS_CODES_TO_RETRY = [502, 503, 504]
```

Comment:
should this go into common.constants?

Reply:
Doesn't look like some common constant for all rucion. I'd keep it here.